### ISSUE TITLE
Implement new design for allowing users to add audio segment labels and descriptions

### ISSUE THREAD

**Issue by brandonhundt** [PRX/publish.prx.org#664]
![Screen Shot 2019-07-02 at 1.05.57 PM.png](https://images.zenhubusercontent.com/5bd1dc3a4cadef67d2695e38/4bacaa53-bc5e-43f4-b18a-da17f4c30116)

**Comment by cavis** [comment]
Is there any case to be made that these fields should be hidden by default?  Maybe some sort of basic/advanced toggle in the top-left header of the box?

For our current user base (podcasters), these fields won't actually do anything.  Meaning they don't end up in the RSS at all... I can imagine getting a lot of questions about them.

**Comment by brandonhundt** [comment]
Yes, there is a case for this. Do we want to add a setting to the series to
show them by default?

**Comment by rpeterman-gp** [comment]
@brandonhundt _Segment Title_ => _Audio Title_ in current audio file type?

**Comment by brandonhundt** [comment]
Correct, what is called the Audio Title on PRI.org will be equivelent to
the Segment title in CMS/Publish.
